include/seqan3/alignment/configuration/align_config_result_type.hpp

```diff
@@ -28,25 +28,24 @@ namespace seqan3::detail
  *
  * \details
  *
- * Implementation of the alignment result capture config.
+ * Implementation of the alignment result type config.
  *
- * \see seqan3::align_cfg::alignment_result_capture
+ * \see seqan3::align_cfg::detail::result_type
  */
 template <typename alignment_result_t>
 //!\cond
     requires is_type_specialisation_of_v<alignment_result_t, alignment_result>
 //!\endcond
-struct alignment_result_capture_element :
-    public pipeable_config_element<alignment_result_capture_element<alignment_result_t>,
+struct result_type_element :
```

Comment:
```suggestion
struct result_type_tag :
```
This should be renamed with the suffix _tag